packages/salesforcedx-vscode-apex-replay-debugger/test/vscode-integration/commands/quickLaunch.test.ts

```diff
@@ -362,7 +363,7 @@ describe('Quick launch apex tests', () => {
     expect(notificationServiceStub.called).to.equal(true);
     const notificationArgs = notificationServiceStub.getCall(0).args;
     expect(notificationArgs[0]).to.equal(
-      "Cannot read property 'length' of undefined"
+      "Cannot read properties of undefined (reading 'length')"
```

Comment:
Swapped to use the version of the error msg that was being reported from the failing test. Seems to be same error, just different string structure